platforms/web/scripts/i18next/parse-translations.ts

```diff
@@ -0,0 +1,66 @@
+import { existsSync, readdirSync, writeFileSync } from 'node:fs';
+import { join } from 'node:path';
+import { execSync } from 'node:child_process';
+
+import prettier from 'prettier';
+
+const i18nextParserConfig = join(process.cwd(), 'i18next-parser.config.js');
+const platformGlob = join(process.cwd(), 'src/**/*.{ts,tsx}');
+
+type WorkspaceEntry = {
+  location: string;
+  workspaceDependencies: string[];
+};
+
+type WorkspaceInfo = Record<string, WorkspaceEntry>;
+
+function syncTranslations() {
```

Comment:
Can you rename this so it doesn't get confused with the `sync-translations` file?

Reply:
Yes, good point